performance-test/blazemeter.yaml

```diff
@@ -13,10 +13,10 @@ scenarios:
     variables:
       PAUSE: 15000
       scheme: https
-      host:
-      username:
-      password:
-      email:
+      host: ivy-web-pre.azure.defra.cloud
```

Comment:
Need to remove the values for these credentials as it's a public repo

host:
username:
password:
email: